src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts

```diff
@@ -284,13 +283,36 @@ export class SnapshotListComponent {
     dialogRef.componentInstance.submit();
 
     dialogRef.componentInstance.success.subscribe((job_res) => {
+      let jobErrors: string[] = [];
+      let jobSuccess: any[] = [];
+
+      job_res.result.forEach((item) => {
+        if(item.error){
+          jobErrors.push(item.error);
+        } else {
+          jobSuccess.push(item.result);
+        }
+      });
+
       dialogRef.close();
       this.entityList.getData();
       this.entityList.selected = [];
 
-      let infoMessage: string = T('Deleted') + ' ' + params[1].length + ' ';
-      infoMessage += params[1].length > 1 ? T('snapshots') : T('snapshot');
-      this.dialogService.Info(infoMessage, '', '320px', 'info', true );
+      
+      if(jobErrors.length > 0){
+        const errorTitle = T('Warning') + ', ' + jobErrors.length + ' of ' + params[1].length + ' ' + T('snapshots could not be deleted.');
```

Comment:
Do we have support for plurals in translation messages?
Different languages may have different order or subject/object.
Something similar to: https://angular.io/guide/i18n#plurals-alternates

Reply:
After discussing with Damian, sounds like this is not something currently supported, so we can skip it for now.